authors/apps/articles/models.py

```diff
@@ -60,6 +60,7 @@ class Article(models.Model):
     disliked_by = models.ManyToManyField(to=settings.AUTH_USER_MODEL,
                                          related_name='disliked_articles',
                                          related_query_name='disliked_article')
+    read_stats = models.IntegerField(default=0)
```

Comment:
I would like to understand the use of this field, and where exactly it is being used.

Reply:
@Karuhanga , thanks for the catch, had planned to use it to store the Article readstats before i changed to a MethodSerializer.